miden-tx/src/verifier/mod.rs

```diff
@@ -58,12 +58,13 @@ impl TransactionVerifier {
 
     // HELPERS
     // --------------------------------------------------------------------------------------------
+
     /// Returns the consumed notes commitment.
-    fn compute_consumed_notes_hash(consumed_notes: &[ConsumedNoteInfo]) -> Digest {
+    fn compute_consumed_notes_hash(consumed_notes: &[Nullifier]) -> Digest {
```

Comment:
Are those again notes to be consumed in a transaction or notes that were already consumed, i.e., the transaction is over?

Reply:
I think these are the same thing - the set of notes to be consumed in a transaction is the same as the set of notes consumed when a transaction is over.